frontends/web/src/components/sidebar/sidebar.css

```diff
@@ -1,3 +1,15 @@
+.closeButton {
+    background: transparent;
+    border: none;
+    display: none;
+    padding: 12px;
```

Comment:
padding is not used for spacing, but to increase the button's touchable / surface area which improves UX (user doesnt have to touch exactly the X button, but surrounding is fine).